tests/data_platform/ingestion/test_raw_row_timestamps.py

```
@@ -0,0 +1,152 @@
+class TestFetchPostsForKeyword:
+    """Tests for fetch_posts_for_keyword()."""
+
+    def test_writes_created_at_and_sync_timestamp(
+        self, monkeypatch: pytest.MonkeyPatch
+    ) -> None:
+        """Bluesky rows include ISO created_at and the run sync_timestamp."""
+        ingestion_params = {"limit": 1, "sort": "latest"}
+        response = mock_search_response([mock_post("at://did:plc:ex/app.bsky.feed.post/a1")])
+        monkeypatch.setattr(
+            "data_platform.ingestion.sync_bluesky._search_posts_page",
+            lambda *args, **kwargs: response,
```

Ruff (0.16.3)

[warning] Unused lambda argument: `args`

(ARG005)